src/mozillavpn.cpp

```diff
@@ -127,6 +127,10 @@ MozillaVPN::MozillaVPN() : m_private(new Private()) {
             setState(StateBackendFailure);
           });
 
+  // TODO: This should trigger a server unavailable modal.
```

Comment:
Created PR https://github.com/mozilla-mobile/mozilla-vpn-client/pull/2538 for launching the modal.